src/templates/services/index.hbs

```diff
@@ -63,13 +66,14 @@
     <section id="below-45">
       <div>
         <div>
-          <h2>Under 45? <span>You're a way off being able to transfer, but we can still assist you.</span></h2>
-          <p>We will help you plan to meet your retirement objectives by:</p>
+          <h2>Under 45 years old? <span>You're a way off being able to transfer but there is still work to be done.</span></h2>
+          <p>Our under 45 service includes:</p>
           <ul>
-            <li>Structuring your finances to allow you to successfully navigate the rules of 2 countries and maximise your outcomes</li>
-            <li>Providing you with retirement projections given the possible different scenarios</li>
-            <li>Determining if your family has any insurance gaps given the tax treatment of death and total &amp; permanent disablement benefits payable from a UK pension</li>
-            <li>Reviewing your estate planning arrangements</li>
+            <li>A complimentary consultation to review your financial plan.</li>
+            <li>Structuring your finances to allow you to successfully navigate the rules of 2 countries and maximise your outcomes.</li>
+            <li>Providing you with retirement projections given the possible different scenarios.</li>
+            <li>Determining if your family has any insurance gaps given the tax treatment of death and total & permanent disablement benefits payable from a UK pension.</li>
```

Comment:
The `&` character is a special one in HTML. So HTML only accepts UTF-8 characters. That means the code you write cannot accept 1. special characters such as double curly quotes, the copyright symbol, special characters used in French, German, Spanish, Japanese... and so on.

However if you do want to show them, you can use these notations in HTML:
http://www.ascii.cl/htmlcodes.htm

So if you want show an ampersand to the user then use `&amp;` or `&#38;` in your HTML.

Note that all these special characters start with &. So in the `<li>` above HTML now thinks a special character is coming... It doesn't expect to present this as an ampersand.